blueprints/module-unification-app/files/package.json

```diff
@@ -24,7 +24,7 @@
     "ember-ajax": "^3.1.0",
     "ember-cli": "github:ember-cli/ember-cli",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^6.16.0",
+    "ember-cli-babel": "^7.1.2"
```

Comment:
Should end with a comma

Reply:
that's clearly 🤦‍♂️ 
Thanks for catching these up - all occurrences fixed